charts/edge-webrtc-service/values.yaml.Example

```diff
@@ -0,0 +1,65 @@
+namespace: "hanwha"
+# fullnameOverride: imagename
```

Comment:
this should be set to the name of the service

Reply:
a few lines down... replica count. You will X number of pods and one IP service for it, but you could use a replicaset instead of a service if you wanted: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/#when-to-use-a-replicaset

Reply:
if you need unique URLs you probably want to put the service as a load balancer rather than ClusterIP (line 26) and you will get a unique url for each service automatically. Probably worth while to nest this template down a level and call it from a higher level with multiple names. less local variables needed that way. The chart.yaml file just needs to have a different name on each to call it. You probably can also put some secrets in on a parent chart and pass them to all the pods you spool